noxfile.py

```diff
@@ -230,3 +271,39 @@ def _get_rust_target() -> str:
 
 
 _HOST_LINE_START = "host: "
+
+_LLVM_COV_COMMAND = [
+    "cargo",
+    "llvm-cov",
+    "--package=pyo3",
+    "--package=pyo3-build-config",
+    "--package=pyo3-macros-backend",
+    "--package=pyo3-macros",
+    "--package=pyo3-ffi",
+]
+
+
+def _cargo_test_package(package: str) -> List[str]:
+    return ["cargo", "test", "--manifest-path", f"{package}/Cargo.toml"]
```

Comment:
Is there a particular reason to use `--manifest-path` instead of `--package` here?

Reply:
Just didn't realise `--package` would work! Have changed to use it 😄